lib/awful/key.lua

```diff
@@ -333,14 +337,36 @@ key.keygroups = {
         {"Right" , "Right" },
         {"Up"    , "Up"    },
         {"Down"  , "Down"  },
-    }
+    },
+    fkeys = {},
+    numpad = {
```

Comment:
A possibility would be to have 2 groups, one with the real characters like this one and one fully indexed?

Reply:
I didn't understand, what are you asking for here? Are you talking about `fkeys` or `numpad`?

What do you mean by "fully indexed"? Do you want me to add the keysyms, like this?

```
        {"#87"   , "KP_End"},
        {"#88"   , "KP_Down"},
        {"#89"   , "KP_Next"},
...
```

(I'll use the same commit to both extend the fkeys to F35 and fix this)